src/main/resources/lang/en.xml

```diff
@@ -366,4 +362,19 @@
         <snowflake>snowflake</snowflake>
         <snowflakes>snowflakes</snowflakes>
     </snowflakes>
+    <death>
```

Comment:
We don't closely follow ocn's pattern elsewhere, I don't see a binding reason to here.